code/API_definitions/device-network-type-subscription.yaml

```diff
@@ -0,0 +1,695 @@
+openapi: 3.0.3
+info:
+  title: Connected Network Type Subscriptions
+  description: |
+    This API provides the customer with the ability to subscribe to notifications on changes to the connected network type of a device.
+
+    # Introduction
+
+    ## Connected Network Type Subscription
+    API consumers can subscribe to receive notifications when the connected network type of a device changes. These notifications are delivered through CloudEvents.
+
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Connected Network Type**: Network Type is intended to provide insight to connected network's capabilities from standards perspective. Actual network capabilities may differ based on implementation and MUST be checked with the connected network provider.
+       - `4GLTE`: if device is connected to the 4G network
+       - `5GNSA`: if device's is connected to the 4G network but using 5G New Radio (NR) capability
+       - `5GSA`: if device is connected to the 5G network
+       - `NON3GPP4G`: if device is connected to the 4G network via WiFi network (e.g. WiFi Calling)
+       - `NON3GPP5G`: if device's is connected to the 5G network via WiFi network (e.g. WiFi Calling)
+       
+    # API Functionality
+
+    The API exposes following capability:
+
+    ## Connected Network Type Subscription Management
+
+    These endpoints allow you to manage subscriptions for receiving notifications on changes to the connected network type of a device.
+    The CAMARA subscription model is detailed in the CAMARA API design guideline document and follows CloudEvents specification.
+
+    **Supported Event Types:**
+      - `org.camaraproject.connected-network-type.v0.network-type-changed`: This event is triggered when the connected network type of the device changes.

```

Comment:
Add the `-subscriptions` on event-type level 
`org.camaraproject.connected-network-type-subscriptions.v0.network-type-changed`